console.rst

```diff
@@ -38,6 +38,19 @@ For example, a command called ``CreateUser`` must follow this structure::
         }
     }
 
+If you're using the :ref:`default services.yml configuration <service-container-services-load-example>`,
```

Comment:
Considering that this is the Console article, this explanation about services may be a bit advanced and/or confusing. Imagine you are a user reading the article: you just want to create and run some commands. You don't even need to know what a service is.

So, why don't we reword the first paragraph to tell the user something like this:

> Symfony registers any PHP class extending from Command as a console command automatically, so you can now execute this command as `.......`.

And keep the mention of services and tags for the `.. note`.